### PR TITLE
bump codemirror-promql and codemirror to the last version

### PR DESCRIPTION
**Pull request by Nexucis**
Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
as a side note, this bump is introducing the support of the trigonometric functions in the UI (like rad, cos ...etc.).
I don't know if you guys already did the bump of the tsdb. So just in case, this PR should be merged once the tsdb has been bumped.

**Comment by Nexucis** [comment]
I think this one is for you @onprem

File: pkg/testutil/e2eutil/sysprocattr.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) The Thanos Authors.
 // Licensed under the Apache License 2.0.
 
+//go:build !linux
```
Comment (onprem) [inline pkg/testutil/e2eutil/sysprocattr.go:4]: This seems unrelated to the current changes.

**Review by onprem — APPROVED** [review]
LGTM, great work as always. Thanks!

I have verified on the current main that the tsdb version we are pulling supports these functions.

File: pkg/testutil/e2eutil/sysprocattr.go (hunk 1 above)
Comment (Nexucis) [reply pkg/testutil/e2eutil/sysprocattr.go:4]: yeah that happens when I ran the command `make assets` it called `make format` and this one is doing this.
Maybe it's a missed when moving to golang v1.17?
